Jenkinsfile

```diff
@@ -55,6 +55,12 @@ pipeline {
             }
         }
 
+        stage('Run Mpt Go Stress Test') {
+            steps {
+                sh 'cd go && go run ./database/mpt/tool stress-test --num-blocks 2000'
+            }
+        }
+
```

Comment:
Since we are already here, maybe we should also move the C++ code format check up. It is not very user friendly to run through all the Go tests, waiting 30+ minutes, to get then a failed test because some formatting is off.

Reply:
ok, good point